visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -642,7 +658,14 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
                     (<Text block variant="small" className={classNames.boldText}>{localization.formatString(localization.WhatIfTab.newProbability, localization.WhatIfTab.loading)}</Text>)}
                 </div>);
         } else {
-            return <div></div>
```

Comment:
shouldn't we still have a default case here in case model type does not match anything and has error?

Reply:
model type is set in the constructor, it will be one of the enumerated values, binary, regression, or multiclass